### ISSUE TITLE
Implement `ChartRevisionSuggester` in WHO GHO dataset import

### ISSUE THREAD

**Issue by bnjmacdonald** [owid/importers#26]
A first version of the WHO GHO dataset import was completed in PR #1 , but this PR only implemented the bulk import for a subset of WHO GHO variables and did not make use of the `ChartRevisionSuggester` to update the associated WHO GHO charts.

Next steps:
- [ ] Add automated chart update code, but only for variables in the `who_gho` namespace.
- [ ] Create a Notion page with a list of datasets/variables outside the who_gho namespace that look like maybe they should be updated by who_gho variables. Then share this list with authors

# Other notes

- For the bulk chart update, it is not immediately clear which charts to update. Of course, any charts using a variable from a dataset under the "who_gho" namespace should be updated, but there are also some charts using variables outside of the who_gho namespace that look like *maybe* they should be updated too.
    - e.g. [MCV1 in WHO GHO](https://owid.cloud/admin/variables/146191) looks like it could reasonably replace [MCV1 in WHO-UNICEF dataset](https://owid.cloud/admin/variables/145109) in corresponding charts. But, [WHO GHO life expectancy](https://owid.cloud/admin/variables/146189) looks like it should definitely NOT replace [Riley, Clio Infra, UN life expectancy](https://owid.cloud/admin/variables/66287) in corresponding charts.
- The challenge is that deciding which of these non-who_gho charts to update with WHO GHO variables will require a fair amount of time to wade through the variables  and make a judgment call.
- Should I restrict the WHO GHO bulk chart update to only the variables already under the who_gho namespace? This is what I lean towards doing, so that I can quickly move on to other bulk imports. That is, unless one of you already has a good sense of which of the non-who_gho datasets should be updated by WHO GHO variables?
    - The answer is "Yes, restrict the WHO GHO bulk chart update to only the variables already under the who_gho namespace for now"

**Comment by edomt** [comment]
(Voluntarily leaving this issue here, since it's directly related to the code in this repo.)

**Comment by bnjmacdonald** [comment]
@spoonerf I've assigned this to you since you've taken over WHO GHO. Can this issue be closed, or is it still relevant to work in progress?

**Comment by spoonerf** [comment]
Let's leave it open for now, I still need to tidy the code a bit and merge it into the main branch on importers. Also, as well as the issues you mentioned above, I think there are some non bulk upload who gho variables and charts floating around that this will remind me to check!

**Comment by spoonerf** [comment]
Update: I imported all of the [WHO GHO variables](https://owid.cloud/admin/datasets/5438). All of the charts in the `who_gho` namespace have been updated. These datasets can now be archived. 

There are a number of non-bulk upload GHO charts where the variables should ideally be replaced with a bulk upload variable to make future updates smoother and ensure we are using the most up-to-date data. 

I will make a list of variables, charts and datasets here: 

- [ ] Archive previous `who_gho` namespace datasets
- [ ] [MCV1 (% 1 year olds immunised)](https://owid.cloud/admin/variables/145109) - replace with https://owid.cloud/admin/variables/146191
